GliaWidgets.xcodeproj/project.pbxproj

```diff
@@ -4532,17 +4536,20 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = YES;
 				CODE_SIGN_ENTITLEMENTS = TestingApp/TestingApp.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = XR3G5NDPNH;
 				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
 				INFOPLIST_FILE = TestingApp/Info.plist;
+				INFOPLIST_KEY_CFBundleDisplayName = "Glia Widgets";
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MARKETING_VERSION = 1.0;
```

Comment:
Why is the marketing version 1.0? Shouldn't it be the same as the current version?

Reply:
Fixed [cf3dbe8](https://github.com/salemove/ios-sdk-widgets/pull/633/commits/cf3dbe8503f4c205815e5226368c9ec97b8c6db1)